### PR TITLE
Formatting fixes

### PR DESCRIPTION
**Pull request by DemiMarie**
This fixes a -Werror=comment failure and avoids problems with git due to CRLF.

### CODE REVIEW

**Comment by bradlitterell** [comment]
@DemiMarie - I can't really tell from github's PR review which files have CRLF problems that you are fixing.  Can you specify exactly what you changed with line endings?  (I recently tried to normalize those and want to understand what was missed.)

File: TPMCmd/tpm/include/GpMacros.h [hunk 1]
```diff
@@ -48,15 +48,19 @@
 //** For Self-test
 // These macros are used in CryptUtil to invoke the incremental self test.
 #if SELF_TEST
-#  define TEST(alg)             \
-    if(TEST_BIT(alg, g_toTest)) \
-    CryptTestAlgorithm(alg, NULL)
+#  define TEST(alg)                    \
+    do                                 \
```
Comment (bradlitterell) [inline TPMCmd/tpm/include/GpMacros.h:52]: Please put curly braces on these constructs. ... 
```
do 
{
...
}
while (0)
```

Thanks

File: TPMCmd/tpm/include/TpmError.h [hunk 2]
```diff
@@ -44,8 +44,8 @@
 #define FATAL_ERROR_CRYPTO           (7)
 #define FATAL_ERROR_NV_UNRECOVERABLE (8)
 #define FATAL_ERROR_REMANUFACTURED      \
```
Comment (bradlitterell) [inline TPMCmd/tpm/include/TpmError.h:46]: Please put the comment above the define.  This is a clang-format glitch.

```
// indicates that the TPM has been re-manufactured after an
// unrecoverable NV error
#define FATAL_ERROR_REMANUFACTURED  (9)
```

File: TPMCmd/tpm/include/Ossl/TpmToOsslHash.h [hunk 3]
```diff
@@ -160,7 +160,7 @@ typedef const BYTE* PCBYTE;
 // Function aliases. The code in CryptHash.c uses the internal designation for the
 // functions. These need to be translated to the function names of the library.
 #  define tpmHashStart_SHA1            \
-    SHA1_Init  // external name of the \
+    SHA1_Init  // external name of the
```
Comment (bradlitterell) [inline TPMCmd/tpm/include/Ossl/TpmToOsslHash.h:163]: clang-format doesn't handle comments on split #defines very well, please move the comment.
This is something I missed when I did the previous format on this branch.

```
// external name of the initialization method
#  define tpmHashData_SHA1           SHA1_Init
```

**Review by bradlitterell — CHANGES_REQUESTED** [review]

File: TPMCmd/Platform/src/NVMem.c [hunk 4]
```diff
@@ -63,7 +63,7 @@ static int NvFileOpen(const char* mode)
 {
 #  if defined(NV_FILE_PATH)
 #    define TO_STRING(s)      TO_STRING_IMPL(s)
-#    define TO_STRING_IMPL(s) #s
+#    define TO_STRING_IMPL(s) #    s
```
Comment (bradlitterell) [inline TPMCmd/Platform/src/NVMem.c:66]: This looks suspicious - the #s should be string-izing the input, and I'm not sure that will happen with the space.

If you ran clang-format.  Would like to know which version?

File: TPMCmd/tpm/include/TpmBuildSwitches.h [hunk 5]
```diff
@@ -61,7 +61,7 @@
 
 // Allow the command line to specify a "profile" file
 #ifdef PROFILE
-#  define PROFILE_QUOTE(a)   #a
+#  define PROFILE_QUOTE(a)   #  a
```
Comment (bradlitterell) [inline TPMCmd/tpm/include/TpmBuildSwitches.h:64]: another case of extra space

File: TPMCmd/tpm/include/TpmASN1.h [hunk 6]
```diff
@@ -85,13 +85,6 @@
 //** Macros
 
 //*** Unmarshaling Macros
-#ifndef VERIFY
```
Comment (bradlitterell) [inline TPMCmd/tpm/include/TpmASN1.h:88]: Why removed?

**Review by bradlitterell — CHANGES_REQUESTED** [review]

File: TPMCmd/Platform/src/NVMem.c (hunk 4 above)
Comment (bradlitterell) [reply TPMCmd/Platform/src/NVMem.c]: Thanks.  This turns out to be a bug in clang-format.
https://github.com/llvm/llvm-project/issues/56255 
Works on 15.0.1.

File: TPMCmd/tpm/include/TpmASN1.h (hunk 6 above)
Comment (DemiMarie) [reply TPMCmd/tpm/include/TpmASN1.h:88]: Is this okay, or should I fix this definition of `VERIFY` instead of removing it?

Comment (bradlitterell) [reply TPMCmd/tpm/include/TpmASN1.h:88]: If it compiles having only one copy in GpMacros, then this is a good find we should leave it out.  When I saw the change I didn't realize it was in the ASN1 header and thought you were deleting it out of GpMacros.

Comment (DemiMarie) [reply TPMCmd/tpm/include/TpmASN1.h:88]: It does indeed.

**Review by bradlitterell — APPROVED** [review]
LGTM, thank you for doing this.

**Comment by DemiMarie** [comment]
You’re welcome @bradlitterell!

**Pull request merged**